powermon/commands/response_definition.py

```diff
@@ -229,6 +239,39 @@ def translate_raw_response(self, raw_value) -> str:
         return int(raw_value.decode())
     
     
+    def response_from_raw_values(self, raw_value) -> list[Response]:
+        value = self.translate_raw_response(raw_value)
+        return [Response(data_name=self.description,
+                        data_value=str(value),
+                        data_unit=self.unit,
+                        device_class=self.device_class,
+                        state_class=self.state_class,
+                        icon=self.icon)]
+    
+    def get_description(self) -> str:
+        return self.description
+      
+# Same INT but base 10
+class ResponseDefinitionTenInt(ResponseDefinition):
+    def __init__(self, index: int, description: str, unit : str,
+                 device_class: str = None,
+                 state_class: str = None,
+                 icon: str = None):
+        self.index = index
+        self.description = description
+        self.unit = unit
+        self.device_class = device_class
+        self.state_class = state_class
+        self.icon = icon
+    
+    def is_valid_response(self, value) -> bool:
+        #Do we need to check if it's negative? Should we have max and min bounds?
+        return isinstance(value, int) / 10
```

Comment:
you are dividing a bool by 10 here and I'm not sure what that will actually return. Currently this method is not used though and will likely be removed from the other response definitions.